### PR TITLE
Document reading/writing data

### PR DESCRIPTION
**Pull request by skddc**
connected to #1003

### CODE REVIEW

File: doc/getting-started/read-and-write-data.rst [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+development. Using :func:`scope`, you can create a new BaseClient scoped to a
+given path::
+
+   const client = remoteStorage.scope('/foo/');
+
+   // List all items in the "foo/" category/folder
+   client.getListing('')
+     .then(listing => console.log(listing));
+
+   // Write some text to "foo/bar.txt"
```
Comment (galfert) [inline doc/getting-started/read-and-write-data.rst]: The example writes to `quotes.txt`, not `bar.txt`.

**Comment by galfert** [comment]
Nice.

Not sure, but maybe it should be noted that for the example to work you also need to claim access to the `foo` category?
Just thinking of people that didn't read the whole "Getting started" guide from the beginning but went directly to this document.

File: doc/getting-started/read-and-write-data.rst (hunk 1 above)
Comment (skddc) [reply doc/getting-started/read-and-write-data.rst]: Good catch, I was changing names back and forth at some point and missed a line.

**Comment by skddc** [comment]
Wouldn't this warn you right in the console?

**Comment by galfert** [comment]
Ah, right. It does.

**Pull request merged**